pkg/apis/core/validation/validation.go

```diff
@@ -2228,6 +2237,19 @@ func ValidatePersistentVolumeClaimSpec(spec *core.PersistentVolumeClaimSpec, fld
 				"must match dataSourceRef"))
 		}
 	}
+	if spec.VolumeAttributesClassName != nil {
+		if opts.EnableVolumeAttributesClass {
+			if len(*spec.VolumeAttributesClassName) == 0 {
+				allErrs = append(allErrs, field.Forbidden(fldPath.Child("volumeAttributesClassName"), "an empty string is disallowed"))
```

Comment:
Forbidden is generally used for permission or policy errors. Invalid is probably a better fitting error type